modules/virt-editing-cdi-cpu-and-memory-defaults.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
+
+[id="virt-editing-cdi-cpu-and-memory-defaults_{context}"]
+= Editing the `CDIConfig` object to override CPU and memory defaults
+
+Modify the default settings for CPU and memory requests and limits for your
+use case by editing the spec attribute of the `CDIConfig` object.
+
+.Procedure
+
+. Use the `oc` client to edit the `cdiconfig/config` and edit the
+`spec: podResourceRequirements` to change the default CPU and memory
+requests/limits.
++
+----
+$ oc edit cdiconfig/config
+----
+
+. Update the `spec` property of the `CDIConfig` object
```

Comment:
I'd rephrase this to contain the info that was originally in the first step (something like "Change the default CPU and memory limits and requests by editing the `spec.podResourceRequirements` fields:")